test/app/features/help_center/data/repositories/audios_repository_test.dart

```diff
@@ -94,7 +95,7 @@ void main() {
       // act
       final result = await sut.requestAccess(audio);
       // assert
-      expect(result, expected);
+      expect(actual, result);
```

Comment:
De acordo com a documentação do `expect`:
> Assert that `actual` matches `matcher`.
>
> [matcher] can be a value in which case it will be wrapped in an [equals] matcher.

Isso dá a entender que o segundo parâmetro é o que é esperado que pode ser utilizado o valor ou algum matcher, sendo assim o primeiro parâmetro (`actual`) seria o valor que se quer validar, então acredito que a ordem ficou trocada e `result` que poderia ser renomeado para `actual`.

Reply:
Atualizado